### ISSUE TITLE
Object detection is not shown up in Apollo.

### ISSUE THREAD

**Issue by kdheejb7** [ApolloAuto/apollo#11805]
We appreciate you go through Apollo documentations and search previous issues before creating an new one. If neither of the sources helped you with your issues, please report the issue using the following form. Please note missing info can delay the response time.

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**: Ubuntu 18.04
- **Apollo installed from (source or binary)**: source
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 5.0

### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**:
`docker/scripts/dev_start.sh
docker/scripts/dev_into.sh
./apollo.sh build_gpu
bootstrap.sh
bridge.sh`

and I started simulator.
`cyber_monitor`

### Supporting materials (screenshots, command lines, code/script snippets):

Hello, 
I'm having trouble in Perception module.
I use the LGSVL, and LGSVL works well but object detection is not displayed in Apollo.

At the module controller tab, I turned on the perception switch.
Sometime Perception switch is turned off automatically, but my GPU capacity is enough.
(I know that there are many issues about this, and the reason is about GPU. I use the 1660 Ti GPU.)
![Screenshot from 2020-07-16 11-30-36](https://user-images.githubusercontent.com/39160667/87619710-cb976400-c757-11ea-8a4f-ad2134142925.png)

Cyber_monitor knows me that perception module is not working normally.
![Screenshot from 2020-07-16 11-27-52](https://user-images.githubusercontent.com/39160667/87619591-7eb38d80-c757-11ea-9079-34f9c3068cfd.png)

What should I do for Perception module working well?

Thank you.

**Comment by ntutangyun** [comment]
are you receiving lidar messages ?

**Comment by kdheejb7** [comment]
No, what means "lidar messages"?

**Comment by kdheejb7** [comment]
> are you receiving lidar messages ?

No, what means "lidar messages"?

**Comment by ntutangyun** [comment]
for obstacle detection to work, you may need **point cloud data** as well sent from LGSVL

**Comment by kdheejb7** [comment]
> for obstacle detection to work, you may need **point cloud data** as well sent from LGSVL

How can I check the point cloud data?
In LGSVL, object detection works well, not in Apollo.

**Comment by lemketron** [comment]
I believe the problem is that your GTX 1660 card is Turing architecture, like the RTX 20x0 series.

Apollo 5.0 does not support Turing, so you'll need to use a card with Pascal architecture like the GTX-1080. Alternatively you could try using Apollo Master, which (up until recently) was working with the Turing architecture, though it is not yet officially supported.  The most recent builds seem to have a problem with camera perception, but if that is resolved then you should be able to switch to apollo-master.  

Or you can check out an older commit of apollo-master such as [this one](https://github.com/ApolloAuto/apollo/commit/eac672b11ae20272f6d91f8eb119bfb942364ef5)

You mentioned using apollo-5.0-lgsvl; there are instructions for using apollo-master with LGSVL [here](https://www.lgsvlsimulator.com/docs/apollo-master-instructions/).

**Comment by daohu527** [comment]
As mentioned above: 
1. check the published message about lidar in cyber_monitor.
2. check the log in “apollo/data/log”  to see if there're some errors.

**Comment by kdheejb7** [comment]
> As mentioned above:
> 
> 1. check the published message about lidar in cyber_monitor.
> 2. check the log in “apollo/data/log”  to see if there're some errors.

![image](https://user-images.githubusercontent.com/39160667/87746318-3bc2ea00-c82b-11ea-9a99-2ec023edc8dc.png)
This is my error message. (apollo/data/log)
I can't find the file 'libperception_component_lidar.so'

**Comment by kdheejb7** [comment]
> I believe the problem is that your GTX 1660 card is Turing architecture, like the RTX 20x0 series.
> 
> Apollo 5.0 does not support Turing, so you'll need to use a card with Pascal architecture like the GTX-1080. Alternatively you could try using Apollo Master, which (up until recently) was working with the Turing architecture, though it is not yet officially supported. The most recent builds seem to have a problem with camera perception, but if that is resolved then you should be able to switch to apollo-master.
> 
> Or you can check out an older commit of apollo-master such as [this one](https://github.com/ApolloAuto/apollo/commit/eac672b11ae20272f6d91f8eb119bfb942364ef5)
> 
> You mentioned using apollo-5.0-lgsvl; there are instructions for using apollo-master with LGSVL [here](https://www.lgsvlsimulator.com/docs/apollo-master-instructions/).

Thank you for advise.
However, Apollo Perception module still did not work well.
There is no different from apollo-5.0/lgsvl.

cyber_launch start modules/transform/launch/static_transform.launch has the error with the message.
![image](https://user-images.githubusercontent.com/39160667/87746588-eb985780-c82b-11ea-9823-d954f1256493.png)

**Comment by daohu527** [comment]
The log you provided just info level logs, you should find some error level logs.

**Comment by kdheejb7** [comment]
Oh, really thank you for informing me the important fact.

I'm trying the apollo-master with LGSVL with your link.



> The log you provided just info level logs, you should find some error level logs.

There is no error log, only INFO log.

**Comment by ntutangyun** [comment]
> ![image](https://user-images.githubusercontent.com/39160667/87746588-eb985780-c82b-11ea-9823-d954f1256493.png)

This error might be due to you have another static_transform process running in the background. 
you may try to close it in the dreamview and then start the transform module in the terminal.

**Comment by kdheejb7** [comment]
> > ![image](https://user-images.githubusercontent.com/39160667/87746588-eb985780-c82b-11ea-9823-d954f1256493.png)
> 
> This error might be due to you have another static_transform process running in the background.
> you may try to close it in the dreamview and then start the transform module in the terminal.

Oh, That's right.
I knew the fact you said, so I solved it just before.
However, I'm still having trouble executing Apollo.

'./apollo.sh test dreamview`
When this commend is executed, there is an error.
![image](https://user-images.githubusercontent.com/39160667/87809460-69914880-c896-11ea-9e8e-2ccb2d74d5f5.png)

![image](https://user-images.githubusercontent.com/39160667/87809563-8ded2500-c896-11ea-84f3-daf0afe709df.png)
![image](https://user-images.githubusercontent.com/39160667/87809591-9b0a1400-c896-11ea-86e2-e0fc10cd7759.png)

and so on...

**Comment by ntutangyun** [comment]
> > 
> 
> How can I check the point cloud data?

is there a channel in cyber monitor related to point_cloud?

**Comment by michael4338** [comment]
> > > 
> > 
> > 
> > How can I check the point cloud data?
> 
> is there a channel in cyber monitor related to point_cloud?

yes, please checkout pointcloud keyword in https://github.com/ApolloAuto/apollo/blob/4d428a30df56f0941f3736c937deb7e33991ffbd/scripts/record_bag.py#L106 for reference

**Comment by kdheejb7** [comment]
> > > 
> > 
> > 
> > How can I check the point cloud data?
> 
> is there a channel in cyber monitor related to point_cloud?

![image](https://user-images.githubusercontent.com/39160667/87847961-072c5c80-c917-11ea-9b87-f2700fb3bdf8.png)


> > > > 
> > > 
> > > 
> > > How can I check the point cloud data?
> > 
> > 
> > is there a channel in cyber monitor related to point_cloud?
> 
> yes, please checkout pointcloud keyword in
> 
> https://github.com/ApolloAuto/apollo/blob/4d428a30df56f0941f3736c937deb7e33991ffbd/scripts/record_bag.py#L106
> 
> for reference
![image](https://user-images.githubusercontent.com/39160667/87847976-13b0b500-c917-11ea-9caa-0ea011ed2948.png)

![image](https://user-images.githubusercontent.com/39160667/87847978-21fed100-c917-11ea-97bf-db19d4a4cecb.png)

Yeah, PointCloud2 turned on now.

**Comment by kdheejb7** [comment]
`cyber_launch start /apollo/modules/perception/production/launch/perception_all.launch'

![image](https://user-images.githubusercontent.com/39160667/87848324-15c84300-c91a-11ea-8e0a-be504725176d.png)


'cyber_launch start /apollo/modules/perception/production/launch/perception_camera.launch`
![image](https://user-images.githubusercontent.com/39160667/87848339-414b2d80-c91a-11ea-94c8-6313deed7785.png)

THis is my error messages :(

**Comment by ntutangyun** [comment]
> 'cyber_launch start /apollo/modules/perception/production/launch/perception_camera.launch`

Hi, I've encountered this yesterday. 

I solved it by specifying the `process_name` attribute in the xml launch file:

modules/perception/production/launch/perception_camera.launch
---
```
<cyber>
    <desc>cyber modules list config</desc>
    <version>1.0.0</version>

    <!-- sample module config, and the files should have relative path like
         ./bin/cyber_launch
         ./bin/mainboard
         ./conf/dag_streaming_0.conf -->

    <module>
        <name>perception_camera</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_camera.dag</dag_conf>
        <!-- if not set, use default process -->
        <process_name>camera_proception</process_name>
        <version>1.0.0</version>
    </module>

    <module>
        <name>motion_service</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_motion_service.dag</dag_conf>
        <!-- if not set, use default process -->
        <process_name>motion_service</process_name>
        <version>1.0.0</version>
    </module>
</cyber>
```
